x-pack/plugins/ml/public/application/routing/routes/explorer.tsx

```diff
@@ -69,25 +71,30 @@ export const explorerRouteFactory = (
   'data-test-subj': 'mlPageAnomalyExplorer',
 });
 
-const PageWrapper: FC<PageProps> = ({ deps }) => {
-  const { context, results } = useResolver(
-    undefined,
-    undefined,
-    deps.config,
-    deps.dataViewsContract,
-    {
-      ...basicResolvers(deps),
-      jobs: mlJobService.loadJobsWrapper,
-      jobsWithTimeRange: () => ml.jobs.jobsWithTimerange(getDateFormatTz()),
-    }
-  );
+const PageWrapper: FC<PageProps> = () => {
+  const {
+    services: {
+      mlServices: { mlApiServices },
+    },
+  } = useMlKibana();
+
+  const { context, results } = useRouteResolver('full', ['canGetJobs'], {
+    getMlNodeCount,
```

Comment:
Can we use basicResolvers() here

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75